src/AppBundle/Entity/Club.php

```diff
@@ -55,98 +54,104 @@ public function __construct()
 
     /**
      * @ORM\OneToMany(targetEntity="AppBundle\Entity\User", mappedBy="club")
-     * @var Collection|User[]
      */
     private $users;
 
+    public function __construct()
+    {
+        $this->created_at = new \DateTime();
+        $this->users = new ArrayCollection();
+    }
 
-
-    public function getId()
+    public function getId(): int
     {
         return $this->id;
     }
 
 
-    public function setName($name)
+    public function setName(string $name)
     {
         $this->name = $name;
 
         return $this;
     }
 
 
-    public function getName()
+    public function getName(): string
     {
         return $this->name;
     }
 
 
-    public function setCity($city)
+    public function setCity(string $city)
     {
         $this->city = $city;
 
         return $this;
     }
 
 
-    public function getCity()
+    public function getCity(): string
     {
         return $this->city;
     }
 
 
-    public function setStreet($street)
+    public function setStreet(string $street)
     {
         $this->street = $street;
 
         return $this;
     }
 
 
-    public function getStreet()
+    public function getStreet(): string
```

Comment:
$street -> nullable=true so it would throw a fatal error, check other variables

Reply:
`?string`